module/help.py

```diff
@@ -0,0 +1,120 @@
+from telegram import Update, InlineKeyboardMarkup, InlineKeyboardButton
+from telegram.ext import CallbackContext
+
+import random
+
+from module.shared import check_log, CUSicon
+
+def help_cmd():
```

Comment:
Maybe this long string could be put in a markdown file, and you could even keep the random emoji with a placeholder, if needed

Reply:
agree!

Reply:
But actually, we can't put in the markdown the randomization of the CUS icons xD

Reply:
Just put a placeholder in the markdown (something like %placeholder%) and do a .replace("%placeholder%, icon)